src/converters/manual_task_converter.ts

```diff
@@ -0,0 +1,117 @@
+import {IIdentity} from '@essential-projects/iam_contracts';
+
+import {ManualTask, ManualTaskList} from '@process-engine/consumer_api_contracts';
+import {
+  IProcessModelFacade,
+  IProcessModelFacadeFactory,
+  IProcessModelService,
+  Model,
+  Runtime,
+} from '@process-engine/process_engine_contracts';
+/**
+ * Used to cache process models during ManualTask conversion.
+ * This helps to avoid repeated queries against the database for the same ProcessModel.
+ */
+type ProcessModelCache = {[processModelId: string]: Model.Types.Process};
+
+export class ManualTaskConverter {
+
+  private _processModelService: IProcessModelService;
+  private _processModelFacadeFactory: IProcessModelFacadeFactory;
+
+  constructor(processModelService: IProcessModelService,
+              processModelFacadeFactory: IProcessModelFacadeFactory) {
+    this._processModelService = processModelService;
+    this._processModelFacadeFactory = processModelFacadeFactory;
+  }
+
+  private get processModelService(): IProcessModelService {
+    return this._processModelService;
+  }
+
+  private get processModelFacadeFactory(): IProcessModelFacadeFactory {
+    return this._processModelFacadeFactory;
+  }
+
+  public async convertManualTasks(identity: IIdentity, suspendedFlowNodes: Array<Runtime.Types.FlowNodeInstance>): Promise<ManualTaskList> {
```

Comment:
```suggestion
  public async convert(identity: IIdentity, suspendedFlowNodes: Array<Runtime.Types.FlowNodeInstance>): Promise<ManualTaskList> {
```

Dann ist: 

ManualTaskConverter.convert()

der Aufruf.